grammar.js

```diff
@@ -2423,6 +2424,15 @@ module.exports = grammar({
       ))
     },
 
+    class_var_assign: $ => {
```

Comment:
Since this is just a specific rule, can you add a note it's only relevant for enum defs?